<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="96" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="96" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
      <class name="training.TestingTest">
        <test-method status="PASS" signature="testmethod5()[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod5" is-config="true" duration-ms="11" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod5 -->
        <test-method status="PASS" signature="testmethod3()[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod3" is-config="true" duration-ms="1" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod3 -->
        <test-method status="FAIL" signature="testmethod(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod" duration-ms="29" started-at="2018-03-13T21:20:35Z" data-provider="Authentication" finished-at="2018-03-13T21:20:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser_1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Juan]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Pedro] but found [Juan]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Pedro] but found [Juan]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at training.TestingTest.testmethod(TestingTest.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod -->
        <test-method status="PASS" signature="testmethod4()[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod4" is-config="true" duration-ms="0" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod4 -->
        <test-method status="PASS" signature="testmethod3()[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod3" is-config="true" duration-ms="0" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod3 -->
        <test-method status="PASS" signature="testmethod(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod" duration-ms="2" started-at="2018-03-13T21:20:35Z" data-provider="Authentication" finished-at="2018-03-13T21:20:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser_2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Pedro]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod -->
        <test-method status="PASS" signature="testmethod4()[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod4" is-config="true" duration-ms="0" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod4 -->
        <test-method status="PASS" signature="testmethod3()[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod3" is-config="true" duration-ms="1" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod3 -->
        <test-method status="PASS" signature="testmethod2()[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod2" duration-ms="1" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod2 -->
        <test-method status="PASS" signature="testmethod4()[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod4" is-config="true" duration-ms="1" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod4 -->
        <test-method status="PASS" signature="testmethod6()[pri:0, instance:training.TestingTest@2a2d45ba]" name="testmethod6" is-config="true" duration-ms="0" started-at="2018-03-13T21:20:35Z" finished-at="2018-03-13T21:20:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod6 -->
      </class> <!-- training.TestingTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
